source/_integrations/buienradar.markdown

```diff
@@ -114,13 +84,63 @@ country_code:
   type: string
 {% endconfiguration %}
 
-### The `name` Variable
+## Sensor
 
-If you specify a name, the camera will get this name as its label. A slugified
-version of the name will be used as the name of the entity.
+The `buienradar` sensor platform will setup separate sensor entities with more detailed weather data. The selected weather station will provide all weather data, with the exception of the forecasted precipitation. The forecasted precipitation data will be retrieved from buienradar using your actual gps-location (and not the location of the nearest weather station).
+
+When camera platform is selected, a window will appear for configuration of additional parameters.
```

Comment:
Shouldn’t “camera” be “sensor” here?

Reply:
This PR needs adjustment, it doesn't match current implementation. I'll markt it draft.